android_prerequisite/README.md

```diff
@@ -2,15 +2,15 @@
 This is the folder to install the Android related eviroment
```

Comment:
In the title of this PR, it will be good to mention the Android version in "for newer android platform".